phalcon/forms/form.zep

```diff
@@ -290,17 +290,19 @@ class Form extends Injectable implements \Countable, \Iterator
 			}
 		}
 
-		let notFailed = true,
-			messages = [];
+		let validationStatus = true;
```

Comment:
In Zephir, all variables used in a given scope must be declared. This process gives important information to the compiler to perform optimizations and validations. This is why tests fails.